stories/index.stories.js

```diff
@@ -0,0 +1,44 @@
+import { html } from "lit";
+import "../stacked-sheets.js";
+
+export default {
+  title: "MyApp",
+  component: "stacked-sheets",
+  argTypes: {
+    title: { control: "text" },
+    counter: { control: "number" },
+    textColor: { control: "color" },
+  },
+};
+
+function Template({ title = "Hello world", counter = 5, textColor, slot }) {
```

Comment:
Missing story for `stacked-sheets-holder`.

Reply:
I have removed storybook. It was not in the original repo and too much of an issue to get working across different versions of LIT.